### ISSUE TITLE
about TorchVOC finetuning results

### ISSUE THREAD

**Issue by lywang76** [ziqi-jin/finetune-anything#40]
Hello, 
I ran your codes on TorchVOC dataset. 

After 200 iterations, i got the following results.
![image](https://github.com/ziqi-jin/finetune-anything/assets/41125918/843f8400-ec78-49aa-bfd1-3d66a72c1ddf)

It seems mIOU is very low. Did you get similar results?
I used your default configuration file.

Thanks,
Liya

**Comment by TimandXiyu** [comment]
I also noticed the convergence is slow enough that there might be some issue with the training config.
I found the mIoU around 20 after 7000 iterations, which is kind of shocking for model like SAM after some intense finetuning.

**Comment by jiaweichaojwc** [comment]
@TimandXiyu 
Hello! Through my tests, I've found that the loss is challenging to decrease, mostly oscillating, and the mIoU is quite low. I feel that the mask adapter approach may not be suitable for fine-tuning. Have you recently tested the code repository, and did you get similar results to mine?
